cli/fathom_web/list.py

```diff
@@ -0,0 +1,32 @@
+from click import argument, command, File, option, Path
+import pathlib
+
+
+@command()
+@option('--out_file', '-o', type=File(mode='w'), default=None,
```

Comment:
If we change this to `--out-file` (with a hyphen), we'll be consistent with fathom-train and the other tools. `click` will change it to an underscore when stuffing its value in a variable.

Reply:
Will do!